testing/files/entrypoint.sh

```diff
@@ -73,7 +73,8 @@ if [[ $overwrite == "y" || $overwrite == "Y" ]]; then
 	if [ "$CHAIN_SPEC" == "testnet" ]; then
 		cp -f testing/networks/80084/*.toml testing/networks/80084/genesis.json ${HOMEDIR}/config
 	else
-		./build/bin/beacond genesis add-premined-deposit --home $HOMEDIR
+		./build/bin/beacond genesis add-premined-deposit --home $HOMEDIR \
+			32000000000 0x20f33ce90a13a4b5e7697e3544c3083b8f8a51d4
```

Comment:
nit: can we start presenting these large decimals with the `_` separators every three digits? Something like `32_000_000_000`. I think it make much easier to read the number and avoid simple mistakes

Reply:
agree in general. Although enforcing a certain standard on CLI may become a hindrance. @chuck-bear would you like to enforce large numbers following the `32_000_000_000` convention (as opposed to just `32000000000`)?